src/lib/vis/uncertainty-cluster-sample-and-mean.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable spaced-comment */
+/* eslint-disable one-var */
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable prefer-const */
+/* eslint-disable func-names */
+/* eslint-disable prefer-template */
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable prefer-spread */
+/* eslint-disable @typescript-eslint/lines-between-class-members */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
[optional]: it’d be great if at least some of these linting issues have been resolved, if possible.

With this many rules silenced at the top of the file, it is hard to feel confident in code quality, even if a lot of testing has been put in place. This especially applies to further changes that are potentially going to be made.

If you do need to disable some of the rules, it’s best to do this line by line or for small sections of the code rather than for the whole file. Here is how this would look:

```ts
// eslint-disable-next-line oops
oops();
```

```ts
/* eslint-disable oops */
oops(1);
oops(2);
oops(3);
/* eslint-enable oops */
```

It’s also very useful to communicate the reason why the rule is disabled – this can be done using `--`:

```ts
// eslint-disable-next-line oops -- Using oops() here is acceptable because so and so
oops();
```

Feel free to do this after merging this PR, if there is value in merging it quickly.

Reply:
Thank you for this comment, the block at the top to do with linting was copied over from another vis function which I used as a basis when developing this. At the moment, similar blocks are present at the top of all [VIS functions](https://github.com/ScottishCovidResponse/rampvis-ui/tree/main/src/lib/vis. Perhaps this is something to look into for all currently implemented VIS functions?

Reply:
I see. In which case we can look into this separately 🙂